assets/app/views/create/fromimage.html

```diff
@@ -1,298 +1,296 @@
+                    <div ng-hide="$parent.expand">
+                      <div>
+                        <label>Replicas: </label>
+                        <span>{{scaling.replicas}}</span>
                       </div>
-                      <div ng-show="$parent.expand">
-                        <label class="number">
-                          Replicas:
-                          <input type="number" class="input-number" min="0" name="replicas" ng-model="scaling.replicas" required ng-pattern="/^\-?\d+$/">
-                        </label>
-                        <div class="has-error" ng-show="form.replicas.$invalid">
-                          <span class="help-block">Replicas must be an integer value greater than or equal to 0</span>
-                        </div>
+                    </div>
+                    <div ng-show="$parent.expand">
+                      <label class="number">
+                        Replicas:
+                        <input type="number" class="input-number" min="0" name="replicas" ng-model="scaling.replicas" required ng-pattern="/^\-?\d+$/">
+                      </label>
+                      <div class="has-error" ng-show="form.replicas.$invalid">
+                        <span class="help-block">Replicas must be an integer value greater than or equal to 0</span>
                       </div>
-                    </osc-form-section>
+                    </div>
+                  </osc-form-section>
 
-                    <!-- /scaling -->
+                  <!-- /scaling -->
 
-                    <label-editor labels="labels" expand="true" can-toggle="false"></label-editor>
-                  </div>
-                  <div class="buttons gutter-top-bottom gutter-top-bottom-2x">
-                    <!-- unable to use form.valid.  need to fix validators in labels and key values directive -->
-                    <button class="btn btn-primary btn-lg"
-                        ng-click="createApp()"
-                        ng-disabled="form.$invalid || nameTaken || disableInputs"
-                        >Create</button>
-                    <a class="btn btn-default btn-lg" href="{{projectName | projectOverviewURL}}">Cancel</a>
-                  </div>
-                </form>
-                <div ng-hide="imageStream">
-                  {{ emptyMessage }}
+                  <label-editor labels="labels" expand="true" can-toggle="false"></label-editor>
+                </div>
+                <div class="buttons gutter-top-bottom gutter-top-bottom-2x">
+                  <!-- unable to use form.valid.  need to fix validators in labels and key values directive -->
+                  <button class="btn btn-primary btn-lg"
+                      ng-click="createApp()"
+                      ng-disabled="form.$invalid || nameTaken || disableInputs"
+                      >Create</button>
+                  <a class="btn btn-default btn-lg" href="{{project | projectOverviewURL}}">Cancel</a>
```

Comment:
isnt project an object here? projectOverviewURL wasnt written to work with an object its expecting the name.  could update projectOverviewURL to work with either the name or the object